index.html

```diff
@@ -67,9 +67,9 @@ <h2>What is it ?</h2>
         <p>Unique Password Builder’s goal is to generate a strong and different password for each website you want to login while still typing the same password (which I call the <strong>master password</strong>) everywhere.</p>
         <p>You can use Unique Password Builder via :
             <ul>
-                <li><a href="https://addons.mozilla.org/en-US/firefox/addon/uniquepasswordbuilder-addon/">Firefox</a> and <a href="https://chrome.google.com/webstore/detail/uniquepasswordbuider/egilgkfibealmbllcigihfhglhipnmie/">Chrome</a> add-ons generating your password that you can then copy and paste,
+                <li><a href="https://addons.mozilla.org/en-US/firefox/addon/uniquepasswordbuilder-addon/">Firefox</a> and <a href="https://chrome.google.com/webstore/detail/uniquepasswordbuider/egilgkfibealmbllcigihfhglhipnmie/">Chrome</a> add-ons generating your password that you can then copy and paste,</li>
                 <li>or a <a href="http://en.wikipedia.org/wiki/Bookmarklet" title="More on bookmarklet">bookmarklet</a> for other browsers that will ask your <strong>master password</strong> and then add a "generate password" link after all password fields.

```

Comment:
Here is another missing </li>
Bug adding </li> is not mandatory by spec so I’m not very convinced about adding them.

Reply:
forget what I said. It’s fine. Thanks